cpp/test/Ice/retry/AllTests.cpp

```diff
@@ -224,6 +224,7 @@ allTests(const Ice::CommunicatorPtr& communicator, const Ice::CommunicatorPtr& c
     {
         cout << "testing invocation timeout and retries... " << flush;
         retry2 = RetryPrx(communicator2, retry1->ice_toString());
+        retry2->ice_getConnection();
```

Comment:
We previously used a checkedCast here, but it was removed when the test was updated to use the new Proxy API. The checkedCast is still used in the C# and Java versions of this test.

In this case, it’s important to ensure that the connection establishment does not occur within the InvocationTimeout, especially when using SSL, where the handshake can take longer than expected.